SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -359,9 +359,20 @@ bool FSpatialGDKEditorToolbarModule::ValidateGeneratedLaunchConfig() const
 		return false;
 	}
 
-	int32 PlayNumberOfServers;
-	GetDefault<ULevelEditorPlaySettings>()->GetPlayNumberOfServers(PlayNumberOfServers);
-	if (!SpatialGDKRuntimeSettings->bEnableHandover && PlayNumberOfServers > 1)
+	const ULevelEditorPlaySettings* LevelEditorPlaySettings = GetDefault<ULevelEditorPlaySettings>();
+
+	bool bLaunchingMultipleServerWorkers = false;
+	if(const int32* ServerCount = LevelEditorPlaySettings->WorkerTypesToLaunch.Find(SpatialConstants::ServerWorkerType))
+	{
+		if (*ServerCount > 1)
+		{
+			bLaunchingMultipleServerWorkers = true;
+		}
+	}
+
+	bLaunchingMultipleServerWorkers = bLaunchingMultipleServerWorkers || LevelEditorPlaySettings->WorkerTypesToLaunch.Num() > 1;
```

Comment:
Entries in `WorkerTypesToLaunch` could have a value of 0 (I think) which would make this flag a false positive. I suggest replacing this and the above conditional with a function that just checks against the sum of all values in `WorkerTypesToLaunch`.